ios/FluentUI/IndeterminateProgressBar/IndeterminateProgressBarTokens.swift

```diff
@@ -9,32 +9,14 @@ import UIKit
 /// Representation of design tokens for the Indeterminate Progress Bar.
 public class IndeterminateProgressBarTokens: ControlTokens {
 
-    /// Creates a new instance of the IndeterminateProgressBarTokens.
-    /// - Parameters:
-    ///   - backgroundColor: Optional override value of the background color.
-    ///   - gradientColor: Optional override value of the gradient color.
-    ///   - height: Optional override value of the height.
-    public init(backgroundColor: DynamicColor? = nil,
-                gradientColor: DynamicColor? = nil,
-                height: CGFloat? = nil) {
-        super.init()
+    weak public var state: MSFIndeterminateProgressBarState?
```

Comment:
unowned?

Reply:
Don't actually need this reference at all!